server/src/main/resources/db/changelog/20170301083925-add-cert-serial-revoked-column.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="20170301083925-1" author="mstead">
+        <comment>
+            Add cp_cert_serial.revoked column to support removing
+            the hibernate formula off of the CertificateSerial entity.
+        </comment>
+        <addColumn tableName="cp_cert_serial">
+            <column name="revoked" type="BOOLEAN" defaultValue="false">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="20170301083925-2" dbms="postgresql,oracle" author="mstead">
```

Comment:
I moved away from the 'property' approach because as it turns out, those are global, and if another changelog happens to define the same property, the value defined in the first processed changelog will be used. I've been bitten by this in the past.

Newer versions of liquibase allows specifying dbms on the sql tag. Would be nice if we could upgrade it at some point -- depends on what RHEL has though.

I'll update it to use properies.